src/map_object.c

```diff
@@ -831,16 +821,16 @@ void sub_0205F148(LocalMapObject* object) {
     }
 }
 
-// ?
-MapObjectManager* sub_0205F160(MapObjectManager* manager) {
+// Likely a scrubbed debug func, but WTF is GF smoking?
```

Comment:
Inappropriate comment

Reply:
huh? there are many such cases of similar comments throughout the codebase, I can change it if you want I suppose